subsys/bluetooth/services/bas/bas_bcs.c

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) 2024 Demant A/S
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/bluetooth/services/bas.h>
+#include <zephyr/bluetooth/gatt.h>
+#include "bas_internal.h"
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_DECLARE(bas, CONFIG_BT_BAS_LOG_LEVEL);
+
+#define BATTERY_CRITICAL_STATUS_CHAR_IDX 9
```

Comment:
For now this is fine, but if you start adding the other optional characteristics, then we can no longer rely on a static index. 

An alternative is to use the UUID when calling `bt_gatt_indicate` which will be futureproof